File: sample-conference-videochat/sample-conference-videochat/Components/CustomSegues/QMRootViewControllerSegue.m
//
//  QMRootViewControllerSegue.m
//  Q-municate
//
//  Created by Igor Alefirenko on 13/02/2014.
//  Copyright (c) 2014 Quickblox. All rights reserved.
//

#import "QMRootViewControllerSegue.h"
#import "AppDelegate.h"

@implementation QMRootViewControllerSegue

- (void)perform {
    
    AppDelegate *delegate = (AppDelegate *)[UIApplication sharedApplication].delegate;
    delegate.window.rootViewController = self.destinationViewController;
}

@end

File: sample-conference-videochat-swift/sample-conference-videochat-swift/sample-conference-videochat-swift/Components/CustomSegues/QMRootViewControllerSegue.swift
//
//  QMRootViewControllerSegue.swift
//  sample-conference-videochat-swift
//
//  Created by Vladimir Nybozhinsky on 04.10.2018.
//  Copyright © 2018 QuickBlox. All rights reserved.
//

import UIKit

class QMRootViewControllerSegue: UIStoryboardSegue {
    override func perform() {
        let delegate = UIApplication.shared.delegate as? AppDelegate
        delegate?.window?.rootViewController = destination
    }
}
